portfolio/src/main/java/com/google/sps/data/BlogHashMap.java

```diff
@@ -71,105 +74,126 @@ public static int getSize(Map<String, LinkedList<BlogMessage>> map) {
   }
 
   // Returns a requested amount of all BlogMessages in map
-  public static LinkedList<BlogMessage> getMessages(Map<String, LinkedList<BlogMessage>> map, int limit) {
+  // Returned values contain the most recent messages.
```

Comment:
Nit: it would be good to give more details about the argument "limit". I am confused here because I am not entirely sure if "limit" here is to limit the size of the returned LinkedList? E.g., if limit=10, does it mean that the returned list will have at most 10 elements? Please be explicit in the comment.

Reply:
Got it